package.json

```diff
@@ -29,6 +29,7 @@
     "babel-preset-env": "1.7.0",
     "babel-register": "6.26.0",
     "bcrypt": "3.0.2",
+    "bluebird": "^3.5.3",
```

Comment:
Lock the version of the "Bluebird" package you installed. See line 32 and line 60